api/server.js

```diff
@@ -13,7 +13,7 @@ const initPassport = require('./passport/init');
 var routes = require('./routes/index')(passport);
 var forgetPasswordRout = require('./routes/forgetPassword');
 var passwordResetRout = require('./routes/reset');
-
+var projectsRoute = require('./routes/project');
```

Comment:
we also need to add authentication and should give the authority to the creator to modify project rout, otherwise, anyone can delete or modify the project.